backend/apis/leads/get-lead.api.js

```diff
@@ -70,7 +70,8 @@ function getLeadById(leadId, cbk, connection) {
         JSON_ARRAYAGG(
           JSON_OBJECT(
             "serviceId", leadServices.serviceId,
-            "serviceName", services.serviceName
+            "serviceName", services.serviceName,
+            "programName", programs.programName
```

Comment:
We should update the [API documentation](https://github.com/JustUtahCoders/comunidades-unidas-internal/blob/master/api-docs/leads.md#response) for this.